holoviews/element/stats.py

```diff
@@ -0,0 +1,125 @@
+import param
+import numpy as np
+
+from ..core.dimension import Dimension, Dimensioned
+from .chart import Chart, Scatter
+
+
+class _StatisticsElement(Chart):
+    """
+    StatisticsElement provides a baseclass for Element types that
+    compute statistics based on the input data. The baseclass
+    overrides standard Dataset methods emulating the existence
+    of the value dimensions.
+    """
+
+    _virtual_vdims = True
+
+    def __init__(self, data, kdims=None, vdims=None, **params):
+        super(_StatisticsElement, self).__init__(data, kdims, vdims, **params)
+        if not self.vdims:
+            self.vdims = [Dimension('Density')]
```

Comment:
Is 'Density' going to be right for all statistical elements?

Reply:
Yeah, this logic isn't right at all, will get to it.